### PR TITLE
Update planets

### PR DESCRIPTION
**Pull request by markBowman**
See https://lcoglobal.redmineup.com/issues/1895

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 [tool.poetry]
 name = "simbad2k"
-version = "1.1.0"
-description = """
```
Comment (markBowman) [inline pyproject.toml:4]: I don't think poetry still supports multi-line strings.

**Review by jnation3406 — APPROVED** [review]
Looks good to me... was it a precision issue, or is this something we will have to update every few years?

**Comment by markBowman** [comment]
> Looks good to me... was it a precision issue, or is this something we will have to update every few years?

It needs updating periodically.

**Review by mgdaily — APPROVED** [review]

**Review by jchate6 — APPROVED** [review]
Silly planets, always moving...

Looks good to me.

**Comment by jchate6** [comment]
> Looks good to me... was it a precision issue, or is this something we will have to update every few years?

The orbital elements are only good for the `"epoch_jd"` (Which was previously April 2018). As the planets orbit the Sun and interact with each other the shape of the orbit changes very slightly and so we need to update them periodically to keep form introducing errors when calculating where they are.

Mercury's orbit also precesses due to General Relativity things and is probably the only one where doing this more often then every few decades would really make a noticeable difference.

**Pull request merged**